README.md

```diff
@@ -76,7 +76,13 @@ This will start the development server. It should automatically open the running
 React app in your default browser, but if it does you can manually open
 [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-### Using the hosted API
+
+
+<!-- 
+Changed the font style to bold to make the Usage area pop a little bit more-->
```

Comment:
❌ can you remove this comment from the actual markdown? It won't provide much value after this PR is closed.

Reply:
Will do.

Reply:
Done